Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -167,12 +167,18 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
     // MARK: - Server Selection
 
-    public var lastSelectedServerInfo: NetworkProtectionServerInfo? {
+    @MainActor
+    private var lastSelectedServer: NetworkProtectionServer? {
```

Comment:
Needed so that we can access the previous `allowedIPs`

Reply:
Also made this a `@MainActor` property, but that change somewhat snowballed and I needed to made a load of functions that access it the same. So open to other suggestions as I might be missing some details of swift concurrency that render this unnecessary.